Analysis/include/SampleDescriptor.h

```diff
@@ -108,7 +108,7 @@ struct SampleDescriptorBase {
         SampleType sampleType;
         double norm_sf{1}, datacard_sf{1}, draw_sf{1};
         std::string cross_section;
-        bool draw{false};
+        bool draw{false}, apply_top_pt_unc{false};
```

Comment:
as far as I see, don't need to define `apply_top_pt_unc` inside `struct Point` - only definition on L129 is currently used in the code.